src/main/java/controllers/FeedPresenter.java

```diff
@@ -18,4 +21,6 @@ public void setFeedFilter(String filter) {
     public void refreshFeed() {
         this.feedManager.refreshFeed();
     }
+
+    public ArrayList<Post> getAllPosts(){return this.feedManager.getPostsFromFeed();}
```

Comment:
Do you think this would fit the new design? I think FeedPresenter would be getting user input using InOut and sending messages to the user using InOut. We might want FeedManager to handle gathering Feed data.